frontend/src/store/slices/userSlice.ts

```diff
@@ -1,9 +1,10 @@
-import { createSlice } from "@reduxjs/toolkit";
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
 import type { PayloadAction } from "@reduxjs/toolkit";
 import type { AppDispatch } from "..";
 import type { Artist } from "../api";
 import { api } from "../api";
 import type { User, UserId, UserSlice } from "./types";
+import { RootState } from "..";
```

Comment:
we should change this to `import type { RootState } from ".."` because in the event the building code doesn't realize its only used as a type, it will cause an import cycle and break down at runtime